src/um/shobjidl.rs

```diff
@@ -17,6 +17,9 @@ use um::shtypes::COMDLG_FILTERSPEC;
 use um::unknwnbase::{IUnknown, IUnknownVtbl};
 use um::winnt::{HRESULT, LPCWSTR, LPWSTR, WCHAR};
 
+DEFINE_GUID!{CLSID_TaskbarList,
+    0x56fdf344, 0xfd6d, 0x11d0, 0x95, 0x8a, 0x00, 0x60, 0x97, 0xc9, 0xa0, 0x90}
```

Comment:
CLSID_TaskbarList should be defined in shobjidl_core.rs

Reply:
@Connicpu so the GUID , ITaskbarList and ITaskbar2List both should added belong to shobjidl_core.rs, not only the GUID, right ?

Reply:
Yes.